packages/core/src/Stores/client-store.js

```diff
@@ -1711,7 +1712,9 @@ export default class ClientStore extends BaseStore {
         if (has_changeable_field) {
             let changeable_fields = [];
             if (changeable && changeable.only_before_auth) {
```

Comment:
Please remove any reference to `changeable_fields` in landing company settings. All profile field edibility logic is centralized in `get_settings->immutable_fields`. Please take `changeable_fields` as deprecated; it will be removed after a while.

Reply:
Thanks for the review. Will do it right away.